test/fuzzer/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ if(NOT fuzzing_coverage)
     # TODO: Aleth reports undefined behaviors, disable it for fuzzing.
     string(REPLACE undefined "" aleth_fuzzing_flags ${fuzzing_flags})
 
-    set(aleth_git_tag v1.7.0-alpha.1)
+    set(aleth_git_tag v1.7.0-rc.1)
```

Comment:
Shouldn't this be matched by bumpversion too? If not, should also fix the bumpversion config.

Reply:
Ah never mind, this is the fixed version it loads.